crates/env/src/api.rs

```diff
@@ -566,10 +566,13 @@ pub fn debug_message(message: &str) {
 /// # Example
 ///
 /// ```
+/// # #[cfg(not(miri))]
+/// # {
```

Comment:
Why's this behind a `not(miri)` flag?

Reply:
These examples call into functions using `asm!` and thus do not compile under Miri

Reply:
Ah, yeah you mentioned that in the PR description, but I didn't see any `asm!` blocks. A short comment explaining that in the code would be useful imo